### PR TITLE
build + remove refresh

### LINKED ISSUE #71: Refactor turning reflect on to not refresh current page
When I turn reflect on, I except my current page to not be refreshed when it the domain is not in my block list. Currently, reflect will reload the page on extension load no matter what

### PR DESCRIPTION
**Pull request by jackyzha0**
some minor refactoring + closes #71

### CODE REVIEW

File: src/background.ts [hunk 1]
```diff
@@ -110,8 +110,13 @@ function turnFilteringOn(): void {
 
 // reloads tab that is currently in focus
 function reloadActive(): void {
-  chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
-    chrome.tabs.reload(tabs[0].id)
+  getStorage().then((storage) => {
```
Comment (googol88) [inline src/background.ts:113]: love the use of the helper functions! kind of wondering what cleanDomain does exactly? Does it just get rid of the `https://` and `www.`?

File: src/content.ts [hunk 2]
```diff
@@ -2,7 +2,6 @@ import BlobAnimation from './blob_animation'
 import { getStorage, logIntentToStorage } from './storage'
 import { cleanDomain } from './util'
 import injectOptionsToOnboarding from './onboarding_options'
```
Comment (googol88) [inline src/content.ts:4]: Kind of curious what was SimpleMDE for?

File: src/content.ts [hunk 3]
```diff
@@ -93,20 +92,21 @@ function iterWhitelist(): void {
...
     setTimeout(() => {
-      loadBlockPage(strippedURL)
+      loadBlockPage()
     }, timeDifference)
   })
 }
 
 // replace current page with reflect block page
-function loadBlockPage(strippedURL: string): void {
+function loadBlockPage(): void {
```
Comment (googol88) [inline src/content.ts:108]: Nice 👍

**Review by googol88 — APPROVED** [review]
lgtm!

File: src/background.ts (hunk 1 above)
Comment (jackyzha0) [reply src/background.ts:113]: yeah just keeps the host (e.g. youtube.com), basically just strips those two you mentioned as well as any sub-path (e.g. youtube.com/channel/whatever)

File: src/content.ts (hunk 2 above)
Comment (jackyzha0) [reply src/content.ts:4]: we were thinking about adding a markdown editor to the main block page for people to take notes but scraped that halfway LOL

**Review by annakvt — COMMENTED** [review]
Lgtm!

**Pull request merged**